### DISCUSSION TITLE
Payment Receipt

### DISCUSSION THREAD

**Discussion by CJStevens17** [sheltermanager/asm3#1488] [Q&A / Help] [answered]
Can I email a Payment Receipt via ASM?

**Comment by bobintetley** [comment, marked as answer]
Use the Receipt/Invoice button on the payment tab of a record or the payment book. You can select one or more payments first to include. There are default "receipt" and "invoice" templates included with SM, which you can edit under Settings->Document Templates.

The receipt can be saved to the person's record (save button after generating from the template) and then emailed from the media tab. Optionally, you can choose to email it as a PDF so that it can't be tampered with.

I'm going to add a ticket to our development list to put an email button on the payment screens so that we can save a step and generate the receipt into the email body in future too.
